pkg/init/init.go

```diff
@@ -221,3 +221,20 @@ func (o *InitClient) PersonalizeName(devfile parser.DevfileObj, flags map[string
 	err := backend.PersonalizeName(devfile, flags)
 	return err
 }
+
+func (o InitClient) PersonalizeDevfileConfig(devfileobj parser.DevfileObj, flags map[string]string, fs filesystem.Filesystem, dir string) error {
+	var backend backend.InitBackend
+	onlyDevfile, err := location.DirContainsOnlyDevfile(fs, dir)
+	if err != nil {
+		return err
+	}
+
+	// Interactive mode since no flags are provided
+	if len(flags) == 0 && !onlyDevfile {
```

Comment:
I'm not sure about these conditions: when I run `odo init` from an empty directory, it first downloads a Devfile (onlyDevfile is then true), then this method is called, and so the `else` branch is selected, which is not what I expected